packages/runtime/container-runtime/src/summary/summarizerNode/summarizerNode.ts

```diff
@@ -398,7 +373,8 @@ export class SummarizerNode implements IRootSummarizerNode {
 	 * Refreshes the latest summary tracked by this node. If we have a pending summary for the given proposal handle,
 	 * it becomes the latest summary. If the current summary is already ahead, we skip the update.
 	 * If the current summary is behind, then we do not refresh.
-	 *
+	 * @param proposalHandle - Handle from the summary ack op
```

Comment:
```suggestion
	 * @param proposalHandle - Handle of the generated / uploaded summary.
```
Note: The handle in summary ack will be different from propsal handle in case of 2 commit summaries